client/platform/web-girder/views/NavigationBar.vue

```diff
@@ -1,39 +1,31 @@
 <script>
 import { mapState } from 'vuex';
-import { all } from '@girder/components/src/components/Job/status';
 
 import NavigationTitle from 'viame-web-common/components/NavigationTitle.vue';
 import UserGuideButton from 'viame-web-common/components/UserGuideButton.vue';
 
+import JobsTab from './JobsTab.vue';
 import { getPathFromLocation } from '../utils';
 
 export default {
   name: 'GenericNavigationBar',
   components: {
     NavigationTitle,
     UserGuideButton,
+    JobsTab,
   },
-  inject: ['girderRest', 'notificationBus'],
+  inject: ['girderRest'],
   data: () => ({
     runningJobIds: [],
   }),
   computed: {
-    ...mapState('Location', ['location']),
+    ...mapState('Location', ['location', 'brandData']),
```

Comment:
brandData isn't under Location.
Think it needs to be updated to have the relevant Brand module,